_layouts/home.html

```diff
@@ -1,6 +1,13 @@
 ---
 layout: "wrapper"
 ---
+<body onload = "myAlert()"/>
```

Comment:
I don't know if this is the best way to do this. 
Should this be in another file and called?
How do we turn this off when covid-19?
Is there a way to make this better?

Reply:
@TheOtherUnknown What do you think?